tests/test_room_resource/test_delete_resource.py

```diff
@@ -32,3 +32,10 @@ def test_non_existant_deleteresource_mutation(self):
             delete_non_existant_resource,
             "Resource not found"
         )
+
+    def test_deleteresource_mutation_admin_different_location(self):
```

Comment:
Make this test function name more descriptive. The name is different from what is being tested